src/workspaces/features/SidebarMenu/SidebarMenu.tsx

```diff
@@ -44,29 +54,58 @@ const SidebarMenu = (props: SidebarMenuProps) => {
     modifiers: POPPER_MODIFIERS,
   });
   const innerMenuRef = useRef<HTMLDivElement>(null);
+
   useOnClickOutside(innerMenuRef, () => {
-    setFalse();
+    if (!isDialogOpen) {
+      // Do not close the menu if the user click in the dialog
+      setFalse();
+    }
   });
+
   useEffect(() => {
     if (isOpen) {
       setFalse();
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [router.asPath]);
 
+  const { data, loading, refetch } = useQuery<
+    WorkspacesPageQuery,
+    WorkspacesPageQueryVariables
+  >(WorkspacesPageDocument, {
+    variables: { page: 1, perPage: 5 },
+    fetchPolicy: "no-cache",
```

Comment:
Not sure about the "no-cache". What does it solve ?

Reply:
I should've remove it. This was during my attempts to load the new list of workspaces after deleting one